pkg/volume/flexvolume/mounter.go

```diff
@@ -47,8 +47,8 @@ func (f *flexVolumeMounter) SetUp(fsGroup *int64) error {
 // SetUpAt creates new directory.
 func (f *flexVolumeMounter) SetUpAt(dir string, fsGroup *int64) error {
 	call := NewDriverCall(f.plugin.getExecutable(), mountCmd, exec.New())
-	call.AppendSpec(f.spec, fsGroup, f.plugin.host)
```

Comment:
ok, will add the result of GetDeviceMountPath; btw if the plugin supports it it could get it itself too so well do an extra driver call... buuuut that's premature optimization. Let's give this src for now.